plugins/amazonq/chat/jetbrains-community/src/software/aws/toolkits/jetbrains/services/cwc/controller/ChatController.kt

```diff
@@ -357,20 +393,20 @@ class ChatController private constructor(
             activeFileContext = activeFileContext,
             userIntent = userIntent,
             triggerType = triggerType,
-            customization = CodeWhispererModelConfigurator.getInstance().activeCustomization(context.project)
+            customization = CodeWhispererModelConfigurator.getInstance().activeCustomization(context.project),
+            relevantTextDocuments = projectContextQueryResult
         )
 
         val sessionInfo = getSessionInfo(tabId)
 
         // Save the request in the history
         sessionInfo.history.add(requestData)
-
         telemetryHelper.recordEnterFocusConversation(tabId)
         telemetryHelper.recordStartConversation(tabId, requestData)
 
         // Send the request to the API and publish the responses back to the UI.
         // This is launched in a scope attached to the sessionInfo so that the Job can be cancelled on a per-session basis.
-        ChatPromptHandler(telemetryHelper).handle(tabId, triggerId, requestData, sessionInfo)
+        ChatPromptHandler(telemetryHelper).handle(tabId, triggerId, requestData, sessionInfo, shouldAddIndexInProgressMessage ?: false)
```

Comment:
nit: do we really need to make `shouldAddIndexInProgressMessage` a nullable? since it be assigned to `false` as default IIRC https://github.com/aws/aws-toolkit-jetbrains/pull/4661/files#diff-db1cf5856c96d1dda59c84c06ddf4d177f083fe80b268da293094eabbe8872ccR129?